www/gatsby-browser.js

```diff
@@ -1,8 +1,16 @@
 const React = require('react');
 const { SettingsContextProvider } = require('./src/context/SettingsContext');
+const { InsightsContextProvider } = require('./src/context/InsightsContext');
+
 
 // wrap whole app in settings context
-exports.wrapRootElement = ({ element }) => <SettingsContextProvider>{element}</SettingsContextProvider>;
+exports.wrapRootElement = ({ element }) => (
+  <SettingsContextProvider>
+    <InsightsContextProvider>
```

Comment:
[curious] Why would these entire documentation site need access to the `InsightsContext`? Should this be isolated to just the "Usage Insights" page/route?

Reply:
I did some investigation and probably there is only way to wrap the whole app instead of just "Usage Insights"